pkgs/by-name/in/intentrace/package.nix

```
@@ -0,0 +1,31 @@
+{
+  fetchFromGitHub,
+  lib,
+  rustPlatform,
+}:
+
+let
+  version = "0.2.4";
+in
+rustPlatform.buildRustPackage {
+  inherit version;
+  pname = "intentrace";
+
+  src = fetchFromGitHub {
+    owner = "sectordistrict";
+    repo = "intentrace";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-Bsis8tL2xahJT/qAFVbbd/CZ7n8KJYLPTIl1a1WHR4c=";
+  };
+
+  cargoHash = "sha256-pyGcQy7p0+Vqv3Khy1hLgahcOpqnbKKmRLZcKwkvVWw=";
+
+  meta = {
+    description = "Prettified Linux syscall tracing tool (like strace)";
+    homepage = "https://github.com/sectordistrict/intentrace";
+    license = lib.licenses.mit;
+    platforms = lib.platforms.linux;
+    mainProgram = "intentrace";
+    maintainers = with lib.maintainers; [ cloudripper ];
```

I'm happy to join you in maintaining this if you'd like

```suggestion
    maintainers = with lib.maintainers; [ cloudripper jk ];
```